python/python/glide/async_commands/core.py

```diff
@@ -670,6 +670,86 @@ async def hkeys(self, key: str) -> List[str]:
         """
         return cast(List[str], await self._execute_command(RequestType.Hkeys, [key]))
 
+    async def hrandfield(self, key: str) -> Optional[str]:
+        """
+        Returns a random field name from the hash value stored at `key`.
+
+        See https://redis.io/commands/hrandfield/ for more details.
+
+        Args:
+            key (str): The key of the hash.
+
+        Returns:
+            Optional[str]: A random field name from the hash stored at `key`.
+            If the hash does not exist or is empty, None will be returned.
```

Comment:
```suggestion
            If the hash does not exist or is empty, `None` will be returned.
```

Reply:
The other docs do not surround None with angled quotes